test/unit/core.js

```diff
@@ -1538,3 +1538,19 @@ testIframeWithCallback( "Don't call window.onready (#14802)", "core/onready.html
 			equal( error, false, "no call to user-defined onready" );
 	}
 );
+
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
It is better to put this `if` inside the test, so amount of tests would be the same for each browser, it would also be easier to filter it out if needed